packages/grafana-toolkit/src/cli/tasks/plugin.ci.ts

```diff
@@ -246,11 +245,11 @@ const testPluginRunner: TaskRunner<PluginCIOptions> = async ({ full }) => {
     },
   };
 
-  const settings = getEndToEndSettings();
+  const settings = plugins.getEndToEndSettings();
   await execa('rimraf', [settings.outputFolder]);
   fs.mkdirSync(settings.outputFolder);
 
-  const tempDir = path.resolve(process.cwd(), 'e2e-temp');
+  const tempDir = path.resolve(process.cwd(), 'e2e/.tmp');
```

Comment:
@ryantxu moved the temp dir under e2e, otherwise @grafana/e2e import in the common tests couldn't be resolved.

Reply:
👍